packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -80,6 +80,9 @@ class OpPerfTelemetry {
 	private connectionStartTime = 0;
 	private gap = 0;
 
+	/**Count of no-ops sent by this client */
```

Comment:
This is not true though. This is only used to telemetry and is reset after the telemetry is sent. 
You should update the name of the property and the comment to reflect that.